tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "compilerOptions": {
     "allowJs": true,
     "target": "es2022",
-    "module": "nodenext",
+    // For now we specifically compile to commonJS for interop with existing libraries
+    // Once fully transitioned to TS, we can switch to nodenext and output ES modules
+    "module": "commonjs",
```

Comment:
This caused me a bit of a headache...

Everything works fine with `nodenext` except for the govuk-frontend imports - because those define both CJS and ESM versions.

TSC (appropriately) determines that the core-front modules should output as CommonJS, but then tries to load the govuk-frontend ones as ESM. This fails, because the ESM imports can't be replicated in CJS. However, if you ask it to specifically use commonjs here, then it all just seems works 🤷

(In future we should be able to change this back and turn the whole thing into ESM)